frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeListsActionsBar/CreateNewCodeListModal/CreateNewCodeListModal.tsx

```diff
@@ -21,45 +18,47 @@ type CreateNewCodeListModalProps = {
   textResources?: TextResource[];
 };
 
-export function CreateNewCodeListModal({
-  onBlurTextResource,
-  onUpdateCodeList,
-  codeListNames,
-  textResources,
-}: CreateNewCodeListModalProps) {
+function CreateNewCodeListModal(
+  {
+    onBlurTextResource,
+    onUpdateCodeList,
+    codeListNames,
+    textResources,
+  }: CreateNewCodeListModalProps,
+  ref: RefObject<HTMLDialogElement>,
+): ReactElement {
   const { t } = useTranslation();
-  const modalRef = createRef<HTMLDialogElement>();
-
   const newCodeList: CodeList = [];
 
-  const handleCloseModal = () => {
-    modalRef.current?.close();
+  const handleCloseDialog = () => {
+    ref.current?.close();
   };
 
   return (
-    <StudioModal.Root>
-      <StudioModal.Trigger variant='secondary'>
-        {t('app_content_library.code_lists.create_new_code_list')}
-      </StudioModal.Trigger>
-      <StudioModal.Dialog
-        ref={modalRef}
-        className={classes.createNewCodeListModal}
-        closeButtonTitle={t('general.close')}
-        heading={t('app_content_library.code_lists.create_new_code_list_modal_title')}
-      >
+    <StudioDialog closedby='any' ref={ref} onClose={handleCloseDialog}>
```

Comment:
The CSS class was lost along the way, making the modal width too slim for creating new code lists 🙂 
```suggestion
    <StudioDialog
      closedby='any'
      ref={ref}
      onClose={handleCloseDialog}
      className={classes.createNewCodeListModal}
    >
```

Reply:
Great spot!